.env.example

```diff
@@ -32,3 +32,6 @@ LOG_LEVEL=
 
 # used when the origin URL of mEditor's host is needed
 MEDITOR_ORIGIN=http://localhost
+
+# Escaped JSON array as a string. E.g., "[{ \"token\":\"dGhpcy1pcy1hLXRva2VuLWZvci1lbmRwb2ludC0x\",\"URL\":\"http://example.com/1\" }]"
```

Comment:
The MVP implementation puts the webhook config (token and URL) in the environment. Future improvements should probably move this into a model (token would be treated as a sensitive field).